src/test/java/de/rabea/game/GameRunnerTest.java

```diff
@@ -41,28 +38,22 @@ public void creates3x3Board() {
         GameRunner gameRunner = new GameRunner(new JavaFXUi(viewUpdaterSpy), new PlayerFactory(null));
         Board board = gameRunner.createBoard(3);
 
-        TestCase.assertEquals(3, board.getDimension());
-    }
-
-    @Test
-    public void creates4x4Board() {
-        GameRunner gameRunner = new GameRunner(new JavaFXUi(viewUpdaterSpy), new PlayerFactory(null));
-        Board board = gameRunner.createBoard(4);
-
-        TestCase.assertEquals(4, board.getDimension());
+        assertEquals(3, board.getDimension());
     }
 
     @Test
     public void showsOptionsForGameMode() {
         GameRunner gameRunner = new GameRunner(new JavaFXUi(viewUpdaterSpy), new PlayerFactory(null));
         gameRunner.displayGameModeOptions();
+
         assertTrue(viewUpdaterSpy.hasShownGameModeOptions);
     }
 
     @Test
-    public void createsANewGame() {
+    public void createsNewGameWithGuiPlayerAndUnbeatablePlayer() {
         GameRunner gameRunner = new GameRunner(new JavaFXUi(viewUpdaterSpy), new PlayerFactory(null));
```

Comment:
What I really want to test here is that it creates a new Game. That's it. But I don't know how to assert on that. Options I'm seeing:
- asserting that game != null (bad... I've been told before never to assert on this)
- check that the game has the correct players (which I'm doing now - but that's not really what I want to test)